plugins/notifications-backend-module-slack/README.md

```diff
@@ -0,0 +1,49 @@
+# @backstage/plugin-notifications-backend-module-slack
+
+The Slack backend module for the notifications plugin.
+
+## Getting Started
```

Comment:
Can we add this as a section in this docs here: https://github.com/backstage/backstage/blob/master/docs/notifications/processors.md